packages/theme-default/src/components/Search/logic/providers/LocalProvider.ts

```diff
@@ -76,8 +79,23 @@ export class LocalProvider implements Provider {
         return cjkWords;
       },
     });
+    this.#cyrilicIndex = FlexSearch.create({
+      ...createOptions,
+      tokenize(str: string) {
```

Comment:
The tokenize method can be abstracted for better maintenance in the future~

Reply:
Thank you for your help in pointing out the shortcomings and teaching me to revise it. It has been fixed.